src/cache/cache_las.c

```diff
@@ -624,11 +631,28 @@ __wt_las_insert_updates(WT_CURSOR *cursor, WT_BTREE *btree, WT_PAGE *page, WT_MU
                 key->data = WT_INSERT_KEY(list->ins);
                 key->size = WT_INSERT_KEY_SIZE(list->ins);
             }
+
             break;
         default:
             WT_ERR(__wt_illegal_value(session, page->type));
         }
 
+        /* Set the first key as the minimum key */
+        if (i == 0)
+            WT_ERR(__wt_buf_set(session, min_las_key, key->data, key->size));
+
+        /*
+         * Keys are sorted in an order, find out whether the current key is minimum or maximum key
```

Comment:
Since the keys are sorted, can we not put the first key as the minimum and the last key as the maximum. Am I missing something here ?

Reply:
corrected.